full_dependencies.conf.json

```diff
@@ -0,0 +1,169 @@
+{
+    // VALUES WE CURRENTLY SPECIFY
+
+    // The version of the config file format.  Do not change, unless
+    // you know what you are doing.
+    "version": 1,
+
+    // The name of the project being benchmarked
+    "project": "PennyLane",
+
+    // The project's homepage
+    "project_url": "pennylane.ai",
+
+    // The URL or local path of the source code repository for the
+    // project being benchmarked
+    "repo": "https://github.com/PennyLaneAI/pennylane.git",
+
+    // The directory (relative to the current directory) that benchmarks are
+    // stored in.  If not provided, defaults to "benchmarks"
+    "benchmark_dir": "benchmarks",
+
+    // The directory (relative to the current directory) to cache the Python
+    // environments in.  If not provided, defaults to "env"
+    "env_dir": ".asv/env",
+
+    // The directory (relative to the current directory) that raw benchmark
+    // results are stored in.  If not provided, defaults to "results".
+    "results_dir": ".asv/results",
+
+    // The directory (relative to the current directory) that the html tree
+    // should be written to.  If not provided, defaults to "html".
+    "html_dir": ".asv/html",
+
+    // The tool to use to create environments.  May be "conda",
+    // "virtualenv" or other value depending on the plugins in use.
+    // If missing or the empty string, the tool will be automatically
+    // determined by looking for tools on the PATH environment
+    // variable.
+    "environment_type": "virtualenv",
+
+    // The matrix of dependencies to test.  Each key is the name of a
+    // package (in PyPI) and the values are version numbers.  An empty
+    // list or empty string indicates to just test against the default
+    // (latest) version. null indicates that the package is to not be
+    // installed. If the package to be tested is only available from
+    // PyPi, and the 'environment_type' is conda, then you can preface
+    // the package name by 'pip+', and the package will be installed via
+    // pip (with all the conda available packages installed first,
+    // followed by the pip installed packages).
+    //
+    "matrix": {
+        "tensorflow": [],
+        "torch": [],
+        "networkx": [],
+        "git+https://github.com/PennyLaneAI/pennylane-lightning.git": [],
+        "git+https://github.com/PennyLaneAI/pennylane-qiskit.git": [],
+        "git+https://github.com/PennyLaneAI/pennylane-cirq.git": [],
+        "git+https://github.com/PennyLaneAI/pennylane-qulacs.git": [],
```

Comment:
Ah was this linked to the question of comparing `master-master` vs `master-release`? I also agree `master-master` probably makes the most sense.